.github/workflows/git-flow.yml

```
@@ -0,0 +1,176 @@
+
+  semantic-pre-release:
+    name: Semantic Pre Release
+    needs: [deploy-check, unit-tests, feature-tests, initialize]
+    permissions:
+      contents: write
+    if: |
+      github.event_name == 'push' &&
+      github.ref == 'refs/heads/develop'
+    uses: VirdocsSoftware/github-actions/.github/actions/semantic-pre-release.yml@ar-CORE-1467-create-sharable-jobs
+    with:
+      semantic-github-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.3)

context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)